getHandler.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const fs = require('fs');
+
+const getHandler = (req, res) => {
+  // console.log(req.method, req.url, req.httpVersion);
+  
+  if (req.url === '/' || req.url === '/index.html') {
+    fs.readFile('./public/index.html', 'utf8', (err, data) => {
+
+      res.write(data);
+      res.end();
+    });
+  } else {
+    fs.readFile(`./public/${req.url.slice(1)}`, 'utf8', (err, data) => {
```

Comment:
Since the filename already starts with a slash, you could just remove the / after the word public.